src/components/shop/ShopResumeMobile.vue

```diff
@@ -0,0 +1,91 @@
+<template>
```

Comment:
The idea is so great but the new component is out of the view port 
![image](https://user-images.githubusercontent.com/35310226/197370721-1d94afe9-7793-4924-9668-a5925fbf90ce.png)

You can make this float and make a sure you test this in multiple screen height

Reply:
I just resolved it